Source/Foundation/bsfUtility/Utility/BsQuadtree.h

```diff
@@ -420,8 +422,7 @@ namespace bs
 
 		private:
 			HNode mCurrentNode;
-			StaticAlloc<Options::MaxDepth * 4 * sizeof(HNode), FreeAlloc> mStackAlloc;
-			StaticVector<HNode, Options::MaxDepth * 4> mNodeStack;
+			std::vector<HNode> mNodeStack;
```

Comment:
We don't use `std::vector`, but `Vector`. Also I avoid dynamic allocations as much as possible because 1. memory fragmentation, 2. Windows debug heap slows the debug mode too much with many of them (and it's useful to have enabled).

Not sure why StaticVector wasn't working, but SmallVector is it's newer and simpler replacement, so I'd prefer that if a change is necessary.

Reply:
ok i'll update.